packages/node_modules/samples/browser-meetings/index.html

```diff
@@ -0,0 +1,367 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Sample: Meetings</title>
+    <style>
+      body {
+        background-color: #eee;
+      }
+
+      section {
+        background-color: #fff;
+        padding: 1rem;
+        margin: 1rem;
+      }
+
+      .flex {
+        display: flex;
+        justify-content: center;
+        flex-wrap: wrap;
+      }
+
+      video {
+        height: 20vh;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>THIS IS A WORK IN PROGRESS SAMPLE</h1>
+    <section>
+      <h2>Meetings</h2>
+      <p>[[APPLICATION DESCRIPTION]]</p>
+    </section>
+
+    <section>
+      <h2>Authentication</h2>
+      <form id="credentials">
+        <fieldset>
+          <legend>Credentials</legend>
+          <input id="access-token" name="accessToken" placeholder="Access Token" value="" type="text" />
+          <input id="access-token-save" title="save" type="submit" value="Save" />
+          <label id="access-token-status">[[CREDENTIALS STATUS]]</label>
+        </fieldset>
+      </form>
+
+      <br />
+
+      <form>
+        <fieldset>
+          <legend>Registration</legend>
+          <input id="registration-register" title="register" type="submit" value="Register" />
+          <input id="registration-unregister" title="unregister" type="submit" value="Unregister" />
+          <label id="registration-status">[[REGISTRATION STATUS]]</label>
+        </fieldset>
+      </form>
+    </section>
+
+    <section>
+      <h2>Meetings Management</h2>
+      <form id="meetings-create">
+        <fieldset>
+          <legend>Create a Meeting</legend>
+          <input id="create-meeting-destination" name="createMeetingDestination" placeholder="Destination" value="" type="text" />
+          <input id="create-meeting-action" title="create" type="submit" value="Create" />
```

Comment:
We should probably add back (some of) the examples
And isn't submit supposed to be a button?

Reply:
`input="submit"` and `<button />` are very similar, and can typically be used interchangeably. I'll switch to button though, for more clarity.